ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/TemplateConfigurationFactory.kt

```diff
@@ -10,37 +10,35 @@ internal object TemplateConfigurationFactory {
     fun create(
         variableDataProvider: VariableDataProvider,
         mode: PaywallViewMode,
-        paywallData: PaywallData,
+        validatedPaywallData: PaywallData,
```

Comment:
I'm wondering if this should keep being `paywallData` to not be confused with `ValidatedPaywallResult`... I think it's ok though.